### PR TITLE
dev-go/delve: bump version to 1.3.2

### PR DESCRIPTION
**Pull request by oz123**
Bug: https://bugs.gentoo.org/694382
Package-Manager: Portage-2.3.76, Repoman-2.3.16
Signed-off-by: Oz Tiram <oz.tiram@gmail.com>

### CODE REVIEW

File: dev-go/delve/delve-1.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+DESCRIPTION="A source-level debugger for the Go programming language"
+HOMEPAGE="https://github.com/go-delve/delve"
+
+LICENSE="MIT BSD BSD-2 Apache-2.0"
+SLOT="0"
+
+RESTRICT="test"
+
+src_compile() {
+	go build -ldflags="-X main.Build=${a3e884e}" -o "${S}/dlv" ./cmd/dlv || die
```
Comment (g4s8) [inline dev-go/delve/delve-1.3.2.ebuild]: @oz123 `main.Build` arg should refer to `6c8c450` commit for `1.3.2` version, see https://github.com/go-delve/delve/releases

**Comment by oz123** [comment]
> @oz123 thanks for this PR. Please check one comment about build version. Also, this PR doesn't fix [origin bug](https://bugs.gentoo.org/show_bug.cgi) fully, since `delve-1.0.0` still having `MIT` license only. I think we can drop `1.0.0` ebuild or update its `LICENSE`, WDYT?

When this PR is merge and we can drop 1.0.0 ... in another PR.

File: dev-go/delve/delve-1.3.2.ebuild (hunk 1 above)
Comment (oz123) [reply dev-go/delve/delve-1.3.2.ebuild]: This is now fixed. 10x!

**Comment by juippis** [comment]
Hmm, seems like the older version got removed, and this doesn't apply cleanly anymore (Manifest). Could you rebase on top of fresh master, and update this PR? I can if you're not willing, but it's a lot more manual work for me than you :P

**Comment by oz123** [comment]
> Hmm, seems like the older version got removed, and this doesn't apply cleanly anymore (Manifest). Could you rebase on top of fresh master, and update this PR? I can if you're not willing, but it's a lot more manual work for me than you :P

No problem. I will do it.

**Pull request closed without merging**